service_unavailable_page/web/public/internal/assets/images/govuk-mask-icon.svg

```diff
@@ -1,7 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="132" height="97" viewBox="0 0 132 97" version="1.1">

```

Comment:
Why does the service unavailable page create so many files?

Reply:
The service unavailable page was designed to be deployed without any dependency on our service, as the service would be down, so it needed its own copy of the GOV.UK Design system.

Reply:
Makes a lot of sense. cheers for the clarification